chart/values.yaml

```diff
@@ -18,6 +18,10 @@ marquez:
     repository: marquezproject/marquez
     tag: 0.12.2
     pullPolicy: IfNotPresent
+  ## Name of the existing secret containing credentials for the Marquez installation.
+  ## When this is specified, it will take precedence over the values configured in the 'db' section.
+  ##
+  existingSecret: marquez-secrets
```

Comment:
Mind if we use `existingSecretName` to be more aligned with the naming `marquez.secretName`?